src/cc65/datatype.c

```diff
@@ -198,6 +225,26 @@ Type* GetImplicitFuncType (void)
 
 
 
+const Type* GetReplacementType (const Type* SType)
```

Comment:
I think that the function name should be more specific: `GetStructReplacementType ()`.

Reply:
The function is also used for unions. Maybe `GetStructOrUnionReplacementType ()` ?

Reply:
There are many *Struct* functions that also work with unions, too. Maybe too many. I'll modify the comments only.